settings.gradle

```diff
@@ -66,6 +66,7 @@ if (file('local.properties').canRead()) {
 
 def calcVersion(buildconfig) {
     def local = gradle.rootProject.findProperty("local")
+    def version = new File(rootDir, 'version.txt').getText().trim()
```

Comment:
I'm not familiar with groovy.. When I went to put this assignment in the outer scope (where we read the `buildconfig`), it resulted in a failure.